### PR TITLE
Kubelet: Add metric for tracking Docker error count

### PR DESCRIPTION
**Pull request by vmarmol**
Will help us monitor error rates.

/cc @yujuhong @dchen1107

### CODE REVIEW

**Comment by dchen1107** [comment]
Like the idea, but you need to run hack/verify-gendocs.sh first :-)

**Comment by vmarmol** [comment]
Runs fine locally :( rebased and re-pushed to see if it fixes Travis.

File: pkg/kubelet/dockertools/instrumented_docker.go [hunk 1]
```diff
@@ -34,130 +34,158 @@ func NewInstrumentedDockerInterface(dockerClient DockerInterface) DockerInterfac
...
 
+// Record the duration of the operation.
+func recordOperation(operation string, start time.Time) {
+	metrics.DockerOperationsLatency.WithLabelValues(operation).Observe(metrics.SinceInMicroseconds(start))
+}
+
+// Record error for metric if an error occured.
+func recordError(operation string, err error) {
+	if err != nil {
+		metrics.DockerErrors.WithLabelValues("list_containers").Inc()
```
Comment (yujuhong) [inline pkg/kubelet/dockertools/instrumented_docker.go]: `WithLabelValues("list_containers")` -> `WithLabelValues(operation)`

Comment (vmarmol) [inline pkg/kubelet/dockertools/instrumented_docker.go]: Ah good catch. Fixed.

(So *thats* why I only got errors for list_containers...)

**Comment by yujuhong** [comment]
LGTM. Just out of curiosity, do we know how much overhead it is to record these numbers?

**Comment by vmarmol** [comment]
@yujuhong for the counters very low, for the latencies a bit more since we have to calculate the distribution, but still small from my understanding.

**Comment by dchen1107** [comment]
LGTM

**Pull request merged**